pkg/maintainer/spv/spv.go

```diff
@@ -1,35 +1,83 @@
 package spv
 
 import (
+	"bytes"
 	"context"
+	"encoding/hex"
+	"fmt"
 	"time"
 
 	"github.com/ipfs/go-log/v2"
 
 	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/maintainer/btcdiff"
+	"github.com/keep-network/keep-core/pkg/tbtc"
 )
 
 var logger = log.Logger("keep-maintainer-spv")
 
+const (
+	// Default value for history depth which is the number of blocks to look
+	// back from the current block when searching for past deposit sweep
+	// proposal submitted events. The value is the approximate number of
+	// Ethereum blocks in a week.
+	spvDefaultHistoryDepth = 40320
```

Comment:
Shouldn't it be 50400 assuming 12s block time? 🤔

Reply:
It should. For some reason I assumed one Ethereum block is about `15s`, but it is indeed close to `12s` (at least in recent months).
Done in 004af961764e3fa51e63aecfa0485b9a32e1990e.